src/components/Footer/Footer.js

```diff
@@ -1,10 +1,14 @@
 import React from "react";
+import GitInfo from 'react-git-info/macro';
+const gitInfo = GitInfo();
 
 const Footer = () => {
   return (
     <footer className="page-footer font-small pt-4">
       <div className="footer-copyright text-center py-3">
         Copyright © 2020 Rezume
+        <br/>
```

Comment:
Can we have the indentation properly? Looks good otherwise.

Reply:
ran the project through the config mentioned.